JustBrewIt/Pages/Index.cshtml

```diff
@@ -52,7 +53,7 @@
                         <div class="carousel-caption text-left">
                             <h2>We are here to HELP.</h2>
                             <p class="text-light font-weight-bolder">Given the COVID-19 outbreak we urge you to stay tuned for the updated guidelines </p><p class="text-light font-weight-bolder">to make informed decisions regarding your travel.</p>
-                            <p><a class="btn btn-lg btn-primary" href="https://www.brewersassociation.org/brewing-industry-updates/covid-19-resource-center/" role="button">See Guidelines</a></p>
+                            <p><a class="btn btn-lg btn-primary" href="https://www.brewersassociation.org/brewing-industry-updates/covid-19-resource-center/" target="_blank" role="button">See Guidelines</a></p>
```

Comment:
Fixed:  I have added command, now if you click link (guidelines) on you home page,  link will open in new tab/window.  
 
Benefit:  User can see your Home page as well as the outside link.  
 
     Reference:    https://www.w3schools.com/html/html_links.asp